app/src/main/java/org/wikipedia/yearinreview/YearInReviewNavigation.kt

```diff
@@ -0,0 +1,6 @@
+package org.wikipedia.yearinreview
+
+enum class YearInReviewNavigation {
+    GetStarted,
```

Comment:
Suggestion: replace `GetStarted` with `Onboarding` which is the one we usually use in the app.